stable/drone/Chart.yaml

```diff
@@ -2,7 +2,7 @@ apiVersion: v1
 name: drone
 home: https://drone.io/
 icon: https://drone.io/apple-touch-icon.png
-version: 2.1.0
+version: 2.1.1
```

Comment:
Maybe bumping the MINOR is more appropriate as this change is not a simple PATCH.

Reply:
Sounds appropriate